Ocelot.Sample.ApiGateway/Controllers/WeatherForecastController.cs

```diff
@@ -0,0 +1,33 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace Ocelot.Sample.ApiGateway.Controllers
```

Comment:
You don't need this controller, remove it

Reply:
done